dali/python/nvidia/dali/plugin/pytorch/torch_utils.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import ctypes
+from nvidia.dali import types
+from nvidia.dali.tensors import TensorListGPU, TensorListCPU, TensorGPU, TensorCPU
+from typing import Union, Any
+
+to_torch_type = {
+    types.DALIDataType.FLOAT: torch.float32,
+    types.DALIDataType.FLOAT64: torch.float64,
+    types.DALIDataType.FLOAT16: torch.float16,
+    types.DALIDataType.UINT8: torch.uint8,
+    types.DALIDataType.INT8: torch.int8,
+    types.DALIDataType.BOOL: torch.bool,
+    types.DALIDataType.INT16: torch.int16,
+    types.DALIDataType.INT32: torch.int32,
+    types.DALIDataType.INT64: torch.int64,
+}
+
+
+def feed_ndarray(
```

Comment:
Once we have docs build, let's make sure that it still [renders properly in the docs](https://docs.nvidia.com/deeplearning/dali/user-guide/docs/plugins/pytorch_plugin_api.html#nvidia.dali.plugin.pytorch.feed_ndarray) after being moved to `torch_utils.py`